pkg/osquery/tables/execparsers/dpkg/parser.go

```diff
@@ -0,0 +1,47 @@
+package dpkg
+
+import (
+	"bufio"
+	//"fmt"
+	"io"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+var allowedKeys = []string{
+	"package",
+	"essential",
+	"priority",
+	"section",
+	"version",
+	"description",
+	"build_essential",
+}
+
+func dpkgParse(reader io.Reader) (any, error) {
```

Comment:
As in the other, could you document the format of what we're parsing to make it easier to reference below and understand what the parsing is doing? The output is a lot more verbose so you don't have to put an entire example, but just mentioning that each package block is separated by newlines and is key-value pairs separated by commas would help.